### DISCUSSION TITLE
help me! Error (3)

### DISCUSSION THREAD

**Discussion by Loofa-of-Doom** [xmake-io/xmake#5205] [Q&A]
I don't have a lot of experience installing things from the console. I have uinstalled and installed a few times. Initially when I opened the app it was looking for xmake.lua so i made one and put it the xmake folder. It then gave me this error even after reinstalling a few times, and removing the .lua file and a .cpp file I had in the xmake folder as well. I downloaded the [xmake-master.win64.exe] version (https://github.com/xmake-io/xmake/releases/download/v2.8.7/xmake-master.win64.exe) ![xmakehelp](https://github.com/xmake-io/xmake/assets/120700234/2a32b5ed-402f-4e88-88d7-892206910952)

**Comment by waruqi** [comment]
please do not run xmake in the xmake install directory.

```console
cd otherdir
xmake create test
cd test
xmake
xmake run
```
